models/club.js

```diff
@@ -2,13 +2,14 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var ClubSchema = new Schema({
-    // club_id would be the sender id of club's facebook account
+    // actual club name
+    club_name: {type: String, unique: true},
+    // club_id would be the sender id of club's facebook account, effectively an owner
     club_id: {type: String, unique: true},
     // will store an array of sender ids, use Member database to find corresponding info
     members: {type: [String]},
```

Comment:
Not needed, would just be in the club's own database.

Might be useful to store each club's mongoDB URI.

Reply:
On second thought, since this URI (and also each page's access token) would need to be queried every time a message is received, it might be best to have them stored as config vars.